software/interface.md

```diff
@@ -61,16 +61,17 @@ This table shows the device revision and which DAPLink Bootloader and Interface
 | 1.3            | 0234       | 0234      | [0234](/docs/software/assets/DAPLink-factory-release/0234_kl26z_microbit_0x8000.hex){: .btn.sm-btn .download}
 | 1.3b           | 0234       | 0241      | [0241](/docs/software/assets/DAPLink-factory-release/0241_kl26z_microbit_0x8000.hex){: .btn.sm-btn .download}
 | 1.5            | 0243       | 0249      | [0249](/docs/software/assets/DAPLink-factory-release/0249_kl26z_microbit_0x8000.hex){: .btn.sm-btn .download}
-| 2.00            | 0255       | 0255      | [0255](/docs/software/assets/DAPLink-factory-release/0255_kl27z_microbit_0x8000.hex){: .btn.sm-btn .download}
-| 2.20            | 0256       | 0256      | [0256](/docs/software/assets/DAPLink-factory-release/nrf52820_microbit_if_crc_5dd23001a7_gcc.hex){: .btn.sm-btn .download}
+| 2.00           | 0255       | 0255      | [0255](/docs/software/assets/DAPLink-factory-release/0255_kl27z_microbit_0x8000.hex){: .btn.sm-btn .download}
+| 2.20           | 0256       | 0256      | [0256](/docs/software/assets/DAPLink-factory-release/0256_nrf52820_microbit_if_crc_5dd23001a7_gcc.hex){: .btn.sm-btn .download}
```

Comment:
I've renamed `nrf52820_microbit_if_crc_5dd23001a7_gcc.hex` to `0256_nrf52820...`, will there be any repercussion with that?
Anything linking to this file directly, like the website?

Reply:
Yes, the website links to this directly so I will need to update that at the same time as we make this live